integreat_cms/static/src/css/style.scss

```diff
@@ -236,6 +235,9 @@ textarea,
         &:focus {
             box-shadow: none;
         }
+        &:read-only {
+            @apply text-gray-500 pointer-events-none cursor-not-allowed;
+        }
```

Comment:
```suggestion
```
Ah, I'm very sorry about my incorrect suggestion, this in fact no needed, since `[type="text"]` is already styled accordingly via the rule above in line 208, thus it does not need to be repeated here.